packages/manager/apps/dedicated/client/app/license/license.controller.js

```diff
@@ -141,6 +142,10 @@ export default /* @ngInject */ (
 
   $scope.resetAction();
 
+  $scope.letterCase = (content) => {
```

Comment:
```suggestion
  $scope.capitalize = capitalize;
```
It avoid creating a new function